packages/devextreme/testing/testcafe/tests/editors/numberBox/mask.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-restricted-syntax */
+import url from '../../../helpers/getPageUrl';
+import createWidget from '../../../helpers/createWidget';
+import {
+  appendElementTo,
+} from '../../../helpers/domUtils';
+import { clearTestPage } from '../../../helpers/clearPage';
+import NumberBox from '../../../model/numberBox';
+
+fixture.disablePageReloads`NumberBox_Mask`
+  .page(url(__dirname, '../../container.html'))
+  .afterEach(async () => clearTestPage());
+
+test('pressing "." should clear previous value (T1199553)', async (t) => {
+  const textBox = new NumberBox('#numberBox');
+  const { input } = textBox;
+
+  await t
+    .click('#numberBox')
+    .pressKey('end shift+left shift+left shift+left shift+left shift+left shift+left shift+left shift+left .')
+    .expect(input.value)
+    .eql('0.00');
+}).before(async () => {
+  await appendElementTo('#container', 'div', 'numberBox', { });
+  await createWidget('dxNumberBox', {
+    format: '#0.00',
+    value: 14500,
+  }, '#numberBox');
+});
```

Comment:
We can write a unit test instead of this:
```javascript
        this.instance.option({
            format: '0#.00',
            value: 123.45
        });

        this.keyboard
            .caret({ start: 0, end: 6 })
            .type('.');

        assert.strictEqual(this.input.val(), '0.00', 'mask value is cleared');
```